.pre-commit-config.yaml

```
@@ -188,7 +188,8 @@ repos:
     name: MyPy, for Python 3.13
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
-    - pytest
+    - pytest # dep of `credential_plugins_test.py`& `importable_test.py`
+    - requests # dep of `credentials.aim`, `credentials.centrify_vault`, `credentials.conjur` & `credentials.hashivault`
```

https://github.com/ansible/awx-plugins/actions/runs/10740379607/job/29788420674?pr=26#step:18:786 revealed that `requests` doesn't have typing information bundled in its dists, apparently. Sometimes, projects have separate type stubs (`*.pyi` files) published as their own PyPI projects. In this case, install them instead, for type-checking purposes:
```suggestion
    - types-requests  # dep of `credentials.aim`, `credentials.centrify_vault`, `credentials.conjur` & `credentials.hashivault`
```

---

Note the annotations that got posted here from GHA — they are coming from the linters and I see they mention needing two spaces before hashes, when adding code comments.

---

Please let me know if I didn't resolve this correctly!

---

Marking this thread as resolved.